config/locales/en.yml

```diff
@@ -1705,38 +1710,41 @@ en:
       tabs:
         summary: Summary
         preview: Preview
+    new:
+      samples_count:
+        zero: "Samples (Count: 0 samples)"
```

Comment:
**nitpick:** Can we reword this to `Samples (X selected)`

Reply:
Fixed in 3d10793efd87aa2ebb37a138a403215178554bbb